contrib/epee/include/epee/profile_tools.h

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2006-2013, Andrey N. Sabelnikov, www.sabelnikov.net
```

Comment:
This file was removed in 109b7b81e4ea11040bc376322b97b81dffb68e42 (merged with the wallet3/RPC PR)